### PR TITLE
Version bump!

### PR DESCRIPTION
**Pull request by meiqimichelle**
New release, hot off the presses: https://github.com/buildkite/test-collector-swift/releases/tag/v0.3.0

I've also got a note from the dev team that "it might be worth including the steps specifically for Xcode from the README" -- if anyone's got time to compare-contrast-update, that would be amaze 🤩 but also -- it is good to just get this version bump out!

### CODE REVIEW

**Review by plaindocs — APPROVED** [review]
Version bump looks good. 
I added the Xcode note, be good to have other eyes on that.

File: pages/test_analytics/swift_collectors.md.erb [hunk 1]
```diff
@@ -12,15 +12,15 @@ You can also upload test results by importing [JSON](/docs/test-analytics/import
 
 Before you start, make sure XCTest runs with access to [CI environment variables](/docs/test-analytics/ci-environments).
 
-1. [Create a test suite](https://buildkite.com/docs/test-analytics), and copy the API token that it gives you.
+1. [Create a test suite](https://buildkite.com/docs/test-analytics), and copy the _Buildkite Test Analytics API token_ you see
```
Comment (ddbeck) [inline pages/test_analytics/swift_collectors.md.erb:15]: style: full stop (obligatory) and trimming for length (optional)

```suggestion
1. [Create a test suite](https://buildkite.com/docs/test-analytics) and copy the _Buildkite Test Analytics API token_.
```

File: pages/test_analytics/swift_collectors.md.erb [hunk 2]
```diff
@@ -12,15 +12,15 @@ You can also upload test results by importing [JSON](/docs/test-analytics/import
...
 
-    To use the Buildkite Test Collector with a Swift Package Manager project, add this repository to the `Package.swift` manifest and add `BuildkiteTestCollector` to any test target requiring analytics:
+1. Add the `test-collector-swift` repository to the `Package.swift` manifest, and add `BuildkiteTestCollector` to any test target requiring analytics:
```
Comment (ddbeck) [inline pages/test_analytics/swift_collectors.md.erb:17]: I realize this didn't originate with this PR, but I wanted to flag something that this change made more apparent to me:

Not being a Swifter (Swifty?), it's not clear to me that this step happens entirely within the manifest file. I suspect this is actually two steps: adding `test-collector-swift` to `Package.swift` and modifying some other file (shown below) to add the collector to the targets. If so, might be nice to put the `Package.swift` bit on its own step.

**Review by karensawrey — APPROVED** [review]

File: pages/test_analytics/swift_collectors.md.erb [hunk 3]
```diff
@@ -35,9 +35,12 @@ Before you start, make sure XCTest runs with access to [CI environment variables
     )
     ```
 
-1. Set the `BUILDKITE_ANALYTICS_TOKEN` secret on your CI to the API token from earlier.
+1. [Securely](/docs/pipelines/secrets) set the `BUILDKITE_ANALYTICS_TOKEN` secret on your CI to the API token from earlier. If you're testing an Xcode project, remember that Xcode doesn't automatically pass environment variables to the test runner, so you need to add them manually. In your test scheme or test plan, go to the _environment variables_ section and add the following key-value pair:
```
Comment (ddbeck) [inline pages/test_analytics/swift_collectors.md.erb:38]: style, optional: semantic line breaks

```suggestion
1. [Securely](/docs/pipelines/secrets) set the `BUILDKITE_ANALYTICS_TOKEN` secret on your CI to the API token from earlier.
If you're testing an Xcode project, remember that Xcode doesn't automatically pass environment variables to the test runner, so you need to add them manually.
In your test scheme or test plan, go to the _environment variables_ section and add the following key-value pair:
```

File: pages/test_analytics/swift_collectors.md.erb [hunk 4]
```diff
@@ -35,9 +35,12 @@ Before you start, make sure XCTest runs with access to [CI environment variables
...
 
-1. Push your changes to a branch, and open a pull request. After a test run has been triggered, results will start appearing in your analytics dashboard.
+    * key: `BUILDKITE_ANALYTICS_TOKEN`
+    * value: `$(BUILDKITE_ANALYTICS_TOKEN)`
+
+1. Push your changes to a branch, and open a pull request. After a test run has been triggered, results will start appearing in your analytics dashboard
```
Comment (ddbeck) [inline pages/test_analytics/swift_collectors.md.erb:43]: style: semantic line breaks (optional), full stop (obligatory), tense consistency (obligatory)

```suggestion
1. Push your changes to a branch, and open a pull request.
After a test run has been triggered, results appear in your analytics dashboard.
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
No showstoppers here but a few little style nits

**Comment by ddbeck** [comment]
Oh darn, this merged just as I submitted my review. I'll open a PR with my suggestions instead.

File: pages/test_analytics/swift_collectors.md.erb (hunk 2 above)
Comment (plaindocs) [reply pages/test_analytics/swift_collectors.md.erb:17]: Me either, but I don't think so. I think **both of these** are in Package.swift, just two sections, *dependencies* and *targets*

**Pull request merged**